tests/gen24bBMP.py

```diff
@@ -410,10 +405,11 @@ def gen_BMP_stream(filename_prefix, golden_filename, num_sink):
             else:
                 pixels[9] += [0, 0, 0]
 
+    string = []
     with open(golden_filename, 'wb') as file:
         for i in range(0, 10):
             saveBMP(filename_prefix + '_' + str(i) + '.bmp', pixels[i], 'RGB', 16, 16)
```

Comment:
FYI , ```autopep8``` changes this line as follows:
```suggestion
            saveBMP(filename_prefix + '_' + str(i) +
                    '.bmp', pixels[i], 'RGB', 16, 16)
```